### PR TITLE
Add duration of audio surveys to sycamore

### PR DESCRIPTION
**Pull request by clementzach**
This PR adds the durations of audio surveys to sycamore.

### CODE REVIEW

**Review by hackdna — CHANGES_REQUESTED** [review]
Thanks!

**Comment by clementzach** [comment]
@hackdna the main issue I'm dealing with is an error in the automated checks. I'm getting:
```
OSError: cannot load library 'libsndfile.so': libsndfile.so: cannot open shared object file: No such file or directory
```

It seems that fixing this is [more complicated](https://stackoverflow.com/questions/57743005/how-to-use-soundfile-on-heroku) than just changing our requirements file. 

Do you know if there's an easy way to solve this, or does this mean that librosa is not "easy to install" and it shouldn't be included as a dependency for forest?

**Comment by clementzach** [comment]
@hackdna thank you so much for figuring that out!

File: forest/sycamore/read_audio.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import os
 from typing import Dict
 
+import librosa  # type: ignore
```
Comment (hackdna) [inline forest/sycamore/read_audio.py:7]: We add modules with missing type hints to https://github.com/onnela-lab/forest/blob/develop/mypy.ini
More details: https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports

**Review by hackdna — CHANGES_REQUESTED** [review]
No problem. Also, please add instructions on how to install `librosa` dependencies in Linux to Sycamore docs: https://forest.beiwe.org/en/latest/sycamore.html

**Comment by clementzach** [comment]
Sorry this slipped my mind, hopefully these changes are sufficient to get this merged before I move on

**Pull request merged**